pkgs/applications/misc/etesync-dav/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, callPackage, python3Packages, radicale2 }:
```

I'm pretty new to NixOS, but had been packaging this myself until I came across your PR. Looks good! :+1:

It seems like `callPackage` isn't used here
 
```suggestion
{ lib, python3Packages, radicale2 }:
```

---

thanks!